microsoft/testsuites/cvm/cvm_boot.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import itertools
+from pathlib import Path
+from typing import Any, Dict, cast
+
+from assertpy.assertpy import assert_that, assert_warn
+
+from lisa import (
+    Logger,
+    Node,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.features.security_profile import (
+    CvmEnabled,
+    SecurityProfile,
+    SecurityProfileSettings,
+)
+from lisa.operating_system import CBLMariner, Posix
+from lisa.sut_orchestrator import AZURE
+from lisa.testsuite import simple_requirement
+from lisa.tools import BootCtl, Lsblk, Reboot, Tpm2
+from lisa.tools.lsblk import PartitionInfo
+from lisa.util import (
+    SkippedException,
+    TcpConnectionException,
+    UnsupportedDistroException,
+    constants,
+)
+from lisa.util.shell import wait_tcp_port_ready
+
+
+@TestSuiteMetadata(
+    area="cvm",
+    category="functional",
+    description="""This test suite covers some common scenarios related to
+    CVM boot on Azure.
+    """,
+)
+class CVMBootTestSuite(TestSuite):
+    def before_case(self, log: Logger, **kwargs: Any) -> None:
```

Comment:
`log` is not used, it doesn't need.

Reply:
pylint will complain without that parameter, so I just keep it there like other places